SPECS/mariner-rpm-macros/macros

```diff
@@ -69,19 +69,26 @@
 %_use_internal_dependency_generator 1
 %_build_id_links alldebug
 
+# Standard brp-macro naming:
+# convert all '-' in basename to '_', add two leading underscores.
+%__brp_compress %{_rpmconfigdir}/brp-compress %{?_prefix}
+%__brp_strip %{_rpmconfigdir}/brp-strip %{__strip}
+%__brp_strip_comment_note %{_rpmconfigdir}/brp-strip-comment-note %{__strip} %{__objdump}
+%__brp_strip_static_archive %{_rpmconfigdir}/brp-strip-static-archive %{__strip}
+%__brp_elfperms %{_rpmconfigdir}/brp-elfperms
+%__brp_remove_la_files %{_rpmconfigdir}/brp-remove-la-files
+
 %__os_install_post_leave_signatures    \
-    %{_rpmconfigdir}/brp-compress \
-    %{_rpmconfigdir}/brp-strip-comment-note %{__strip} %{__objdump} \
-    %{_rpmconfigdir}/brp-strip-static-archive %{__strip} \
+    %{?__brp_compress} \
+    %{?__brp_strip_comment_note} \
+    %{?__brp_strip_static_archive} \
     %{?py_auto_byte_compile:%{?__brp_python_bytecompile}} \
     find %{buildroot} -name "*.pc" | xargs -I{} sed -i -e 's@-Wl,-dT,%{_topdir}/BUILD/module_info.ld@ @' {} \
     %{nil}
 
 %__os_install_post    \
     %{__os_install_post_leave_signatures} \
-    %{_rpmconfigdir}/brp-strip %{__strip} \
-    %{_rpmconfigdir}/brp-strip-debug-symbols %{__strip} \
```

Comment:
Note: removed brp-elfperms due to error seen building rpms with modified toolchain:

```
chroot: failed to run command ‘rpm’: Permission denied 
```

Reply:
upstream __os_install_post for reference: https://github.com/rpm-software-management/rpm/blob/8ec9cfc81c3f8326a1c7ee5b795ef517cd48e6b4/platform.in#L64

Reply:
Curious... I'm a bit fuzzy as to what the impact is of us not having this script run, but seems like something that could be deferred?

Reply:
It's something we never had before, so should be fine to leave out. From the upstream commit, it looks like an optimization to reduce the amount of binaries scanned for dependency generation by removing the execute bit. We can probably evaluate later on, but for now it definitely causes some problems. Upstream commit: https://github.com/rpm-software-management/rpm/commit/80818e4f902ba3cf85e4cfcd8a7a4c71c601f3cf